documentation/latest/dev/guide.md

```diff
@@ -3,24 +3,32 @@ layout: documentation
 title: Developer guide
 ---
 
+## Integration tests
 
-# Integration tests
+### Quick start
 
-## Quick start
+#### Execute integration tests locally
+
+To execute integration tests locally, you need to first start the
```

Comment:
Pls add a section on the steps needed to test a workflow that is delivered by workflow-example projects.
What I'd like to describe is the workflow-example need to be developed, packaged and built and requires the service to be restart with it.
In kind it requires rebuilding the image and repeating the steps (build/push to kind/...).

Reply:
Added a section in this [commit](https://github.com/parodos-dev/parodos-dev.github.io/pull/11/commits/7829140aa688e920f0d812d2f2ecf16b3b2c5c11). I'm not so sure that it is ok, so any additional comment is appreciated.